src/wazuh_modules/vulnerability_scanner/src/policyManager/policyManager.hpp

```diff
@@ -139,6 +139,7 @@ class PolicyManager final : public Singleton<PolicyManager>
             newPolicy["updater"]["ondemand"] = true;
             newPolicy["updater"]["topicName"] = "vulnerability_feed_manager";
             newPolicy["updater"]["configData"] = nlohmann::json::object();
+            newPolicy["updater"]["configData"]["agentName"] = "Wazuh VulnerabilityDetector";
```

Comment:
I think `agentName` might be a little confusing here.
How about something like `clientModule`, `clientName`, `consumerName` (though this could be confused with the CTI consumers)?

Reply:
I had this in mind... I believe `consumerName` is appropriate! Renamed.